packages/amplify-cli/src/__tests__/commands/env.test.ts

```diff
@@ -2,15 +2,25 @@ describe('amplify env: ', () => {
   const mockExit = jest.fn();
   jest.mock('amplify-cli-core', () => ({
     exitOnNextTick: mockExit,
+    pathManager: { getAmplifyMetaFilePath: jest.fn().mockReturnValue('test_file_does_not_exist') },
   }));
   const { run: runEnvCmd } = require('../../commands/env');
+  const { run: runAddEnvCmd } = require('../../commands/env/add');
   const envList = require('../../commands/env/list');
   jest.mock('../../commands/env/list');
 
   it('env run method should exist', () => {
     expect(runEnvCmd).toBeDefined();
   });
 
+  it('env add method should exist', () => {
+    expect(runAddEnvCmd).toBeDefined();
+  });
+
+  it('env add method should throw if meta file does not exist', () => {
+    expect(async () => await runAddEnvCmd()).rejects.toThrow();
```

Comment:
Can you assert on the error message so we know we're running into the correct error.